package/MDAnalysis/lib/util.py

```diff
@@ -295,7 +295,7 @@ def anyopen(datasource, mode='r', reset=True):
                     stream.reset()
                 except (AttributeError, IOError):
                     try:
-                        stream.seek(0L)
```

Comment:
Though if it's just seeking 0, then I don't see why you need the 'L' there though, so this is fine